### PR TITLE
install vite, chakra ui, wagmi

### PR DESCRIPTION
**Pull request by 0xvgd**
https://www.loom.com/share/f2075a5e8a2a4f4f9e826aca6997d99b

### CODE REVIEW

**Comment by polymorpher** [comment]
feel free to continue - let me know when initial version is ready for testing and please keep it simple

File: src/main.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  autoConnect: true,
+  provider,
+  webSocketProvider,
+})
+
+ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
+  <React.StrictMode>
+    <ChakraProvider>
+      <WagmiConfig client={client}>
+        <Container my="5">
```
Comment (polymorpher) [inline src/main.tsx:24]: what is my=5?

File: src/config.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { Abi } from 'abitype'
+import erc721ABI from '~/abis/erc721ABI.json'
+import NameWrapper from '~/abis/NameWrapper.json'
+import { ContractConfig } from './types'
+
+export const nameWrapperContract: ContractConfig = {
+  address: "0x69e756D56762Fc66ADE2eA8Da4cEd4F888d0cf8A",
```
Comment (polymorpher) [inline src/config.ts]: make these addresses configurable with env?

File: src/App.tsx [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+      if (owner === CONFIG.nameWrapperContract.address) {
+        const wrappedOwner = (await readContract({
+          ...CONFIG.nameWrapperContract,
+          functionName: "ownerOf",
+          args: [getWrappedTokenId(domain)],
+        })) as string
+
+        setWrappedOwner(wrappedOwner)
+      }
+    } catch {}
```
Comment (polymorpher) [inline src/App.tsx]: could have network error here? Maybe show some error popup and print to console

**Comment by polymorpher** [comment]
Code looks good. Will test later today. Have you tested with your domains? Can you post a video?

File: src/main.tsx (hunk 1 above)
Comment (0xvgd) [reply src/main.tsx:24]: margin vertically (margin top & margin bottom)

**Comment by polymorpher** [comment]
good now, need to add support for multiple wallet connectors, polish ui, show tx link in notifications, add more info (collection, tokenid, data urls), add some links (to domain itself, marketplaces, explorer page of the nft and collection, and maybe more)

**Pull request merged**